pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json.template

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "Microsoft.Net.Native.Compiler": "1.6.0",
+    "Microsoft.Private.CoreFx.UAP": "4.4.0-preview1-25205-01",
```

Comment:
We should probably diff the file list before and after this change of the meta-package to ensure there are no missing files (it is of course expected there will be new files).